index.html

```diff
@@ -0,0 +1,87 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <title>Door Status</title>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    
+    <script>
+     function doSetup(){
+         $('#button').click(function (event){
+             if($('#open').val() == 1){
+                 state_string = "Unlocked";
+             }else{
+                 state_string = "Locked";
+             };
+             $.ajax({
+                    url: 'https://door.stuvus.uni-stuttgart.de/castle/lock',   
+                    type: 'PUT',  
+                    dataType: "json",
+                    data: JSON.stringify( {state: state_string}),
+                    success: true 
+                })
+                ;
+         });
+     };
+     doSetup();
+     function doPoll(){
+         $.get('https://door.stuvus.uni-stuttgart.de/castle/lock', function(data, status){
+	     $('#status').text(data.state);
+	     if(data.state == "Locked"){
+	     	$('#status').removeClass("btn-success").addClass("btn-danger");
+	     	$('#button').text("Open Door");
+	     	$('#open').val("1");
+		$('#image').attr("src","/img/stuvus-icon-locked.png");
+	     }else{
+	     	$('#status').removeClass("btn-danger").addClass("btn-success");
+	     	$('#button').text("Lock Door");
+	     	$('#open').val("0");
+		$('#image').attr("src","/img/stuvus-icon-open.png");
+	     }
+
+             setTimeout(doPoll,0500);
```

Comment:
Why is this octal? And so badly idented? And why are there never spaces after commas in function calls?